spaceward/src/features/modals/ApproveActionModal.tsx

```
@@ -10,10 +10,11 @@ import { ModalParams } from "./types";
 import { useModalState } from "./state";
 import clsx from "clsx";
 import { usePublicClient } from "wagmi";
+import { env } from "@/env";
 
 export default function ApproveModal({ hidden }: ModalParams<{}>) {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)